src/python/WMCore/MicroService/MSTransferor/MSTransferor.py

```diff
@@ -676,3 +690,132 @@ def _diskPNNs(self, pnnList):
             else:
                 diskPNNs.add(pnn)
         return diskPNNs
+
+    def updateSites(self, doc):
+        """
+        Update sites API provides asynchronous update of Site info.
+
+        :param doc: JSON payload with the following data structures:
```

Comment:
From the source code, it looks like we only save the workflow name. I think that is correct, but we then need to update this docstring.

Reply:
We receive this record `[{'workflow': <wflow name>, 'SiteWhiteList' ['T1', ...], 'SiteBlackList': ['T2',...]}, {...}]` from upstream and this is what is saved into a file with workflow name as a file name. This allows to keep site lists when we need to run business logic and avoid extra calls to upstream service.

Reply:
Based on the latest discussions, you probably want to receive only the workflow name as payload in the REST API.